spec/controllers/wishlists_controller_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "rails_helper"
+
+RSpec.describe WishlistsController do
+  let(:valid_parameters) { { name: "Foo", description: "Foo" } }
+  let(:invalid_parameters) { { name: "", description: "" } }
+
+  describe "GET #new" do
+    before do
+      request_get
+    end
+
+    it "returns 200 HTTP status code" do
+      expect(response).to have_http_status(200)
+    end
+
+    it "renders the 'new' template" do
+      expect(response).to render_template(:new)
+    end
+
+    it "assigns a new wishlist as @wishlist" do
+      expect(assigns(:wishlist)).to be_a_new(Wishlist)
+    end
+  end
+
+  describe "POST #create" do
+    context "with valid parameters" do
+      before do
+        request_post(valid_parameters)
+      end
+
+      it "creates a new Wishlist" do
+        expect {
+          post :create, params: { wishlist: valid_parameters }
+        }.to change(Wishlist, :count).by(1)
+      end
+
+      it "assigns a newly created wishlist as @wishlist" do
+        expect(assigns(:wishlist)).to be_a(Wishlist)
+        expect(assigns(:wishlist)).to be_persisted
+      end
+
+      it "redirects to the form page" do
+        expect(response).to redirect_to(new_wishlist_path)
+      end
+    end
+
+    context "with invalid parameters" do
+      before do
+        request_post(invalid_parameters)
+      end
+
+      it "returns 200 HTTP status code" do
+        expect(response).to have_http_status(200)
+      end
+
+      it "assigns a newly created but unsaved wishlist as @wishlist" do
+        expect(assigns(:wishlist)).to be_a_new(Wishlist)
+      end
+
+      it "doesn't create new wishlist" do
+        expect {
+          post :create, params: { wishlist: invalid_parameters }
+        }.to change(Wishlist, :count).by(0)
```

Comment:
And I think the `.not_to change(Wishlist, :count)` syntax would work here better.